src/src/Helpers/_Plugin_SensorTypeHelper.cpp

```diff
@@ -41,7 +41,8 @@ void sensorTypeHelper_webformLoad_allTypes(struct EventStruct *event, int pconfi
     static_cast<uint8_t>(Sensor_VType::SENSOR_TYPE_DOUBLE_SINGLE),
     static_cast<uint8_t>(Sensor_VType::SENSOR_TYPE_DOUBLE_DUAL),
 #endif
-    static_cast<uint8_t>(Sensor_VType::SENSOR_TYPE_STRING)
+    static_cast<uint8_t>(Sensor_VType::SENSOR_TYPE_STRING),
+    static_cast<uint8_t>(Sensor_VType::SENSOR_TYPE_EASYFETCH)
```

Comment:
Extending lists like these should add a comma after the last item, to next time avoid having a git-change of 3 lines, instead of just adding 1 line. 😃